static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -37,39 +37,52 @@ const submitPartialFormStatus = (
     });
 };
 
+const getNewsletterHtmlPreference = (): Promise<string> =>
+    fastdom.read(() => {
+        if ($('[name="htmlPreference"]:checked').val()) {
+            return $('[name="htmlPreference"]:checked').val();
+        } else if ($('[name="htmlPreference"]').val()) {
+            return $('[name="htmlPreference"]').val();
```

Comment:
Since these selectors are very expensive they should be scoped to the form, if possible.

Reply:
agreed. at the moment i wanted to err on the side of caution because not finding them breaks the form but mario is working on making the field optional which (EVEN BETTER) will remove this whole query altogether!